src/components/SignupModal/SignupModal.jsx

```diff
@@ -0,0 +1,26 @@
+import "../../pages/auth/auth.scss";
+import {Modal,InputBox} from "../../components/index";
+export const SignupModal=({isOpen,onClose})=>{
+    return(
+        <Modal isOpen={isOpen} onClose={onClose}>
+             <div className="px-0-25 flex-col flex-center">
+        <div className="logo">
+          <img src="../Assets/logosm.png"/>
+        </div>
+        <h2 className="login-page__login-title">Signup</h2>
+        {/* <div className="error-msg mt-1 px-0-75 py-0-5">
+          Error in Signup
+        </div> */}
```

Comment:
You can remove the comment from here if its not required in future.

Reply:
will remove it after further implementation